pom.xml

```diff
@@ -50,9 +50,23 @@
     <nativeLibOnlyDir>${project.build.directory}/native-lib-only</nativeLibOnlyDir>
     <nativeJarWorkdir>${project.build.directory}/native-jar-work</nativeJarWorkdir>
     <aprVersion>1.5.2</aprVersion>
+    <aprMd5>98492e965963f852ab29f9e61b2ad700</aprMd5>
     <boringsslBranch>chromium-stable</boringsslBranch>
     <libresslVersion>2.3.4</libresslVersion>
+    <!--
+        NB: libressl does not currently publish sha256 signatures and instead relies on signify
+        to verify releases. The project does not have a securely published GPG key.
+
+        Verifying a libressl release:
+        - Download the new version and the SHA256.sig from http://ftp.openbsd.org/pub/OpenBSD/LibreSSL/
+        - Grab the signify public key from https://www.openbsd.org/libressl/signing.html if you
+          don't already have it
+        - Verify the release: signify -V -x SHA256.sig  -p libressl.pub -m libressl-{libresslVersion}.tar.gz -e
```

Comment:
actually, i need to read up more on signify: this appears not actually validate the signature...sigh